skins/cat17/address-form/package.json

```diff
@@ -20,13 +22,32 @@
     "@vue/cli-service": "^3.3.0",
     "@vue/eslint-config-prettier": "^4.0.1",
     "@vue/eslint-config-typescript": "^3.2.0",
+    "@vue/test-utils": "^1.0.0-beta.25",
     "babel-eslint": "^10.0.1",
     "eslint": "^5.8.0",
     "eslint-plugin-vue": "^5.0.0",
+    "jest": "^23.4.2",
     "node-sass": "^4.9.0",
     "sass-loader": "^7.0.1",
     "terser": "~3.14",
+    "ts-jest": "^23.10.5",
     "typescript": "~3.1.1",
+    "vue-jest": "^3.0.2",
     "vue-template-compiler": "^2.5.21"
+  },
+  "jest": {
+    "moduleFileExtensions": [
+      "js",
+      "ts",
+      "json",
+      "vue"
+    ],
+    "transform": {
+      ".*\\.(vue)$": "vue-jest",
+      "^.+\\.tsx?$": "ts-jest"
+    },
+    "testRegex": "(/__tests__/.*|(\\.|/)(test|spec))\\.(jsx?|tsx?)$",
+    "verbose": true,
+    "testURL": "http://localhost/"
```

Comment:
Do you have personal preferences about having the jest config in `package.json` instead of `jest.config.js`? I slightly prefer separate files for the following reasons:
* package.json is as small as possible
* package.json contains only npm-related stuff
* easier to jump to in the IDE and easier  to find in general.

Having so many config files( tsconfig, babel config, es/tslint config, webpack config, etc. etc.) is also not ideal, but I think the benefits outweigh the drawbacks.

Reply:
Indeed we have too many config files but I agree it is more sane to keep `package.json` clean and store only npm-related things there.